### PR TITLE
Home page CSS changes

### PR DESCRIPTION
**Pull request by Kani1996**
image

### CODE REVIEW

File: med-inventory/src/layout/NavBar.jsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ import Typography from "@material-ui/core/Typography";
 import Divider from "@material-ui/core/Divider";
 import IconButton from "@material-ui/core/IconButton";
 import MenuIcon from "@material-ui/icons/Menu";
+import HelpIcon from "@material-ui/icons/Help";
```
Comment (KarthikKumarDev) [inline med-inventory/src/layout/NavBar.jsx:12]: Mention all the changes done in description. Keeping commits atomic will help things in long run. 
1. Logo Change.
2. Help Icon.
3. Other change....

File: med-inventory/src/layout/NavBar.jsx [hunk 2]
```diff
@@ -30,6 +31,7 @@ import UploadData from "../components/upload-data/UploadData";
 import { Router } from "@reach/router";
 import { Link } from "@reach/router";
 
+
```
Comment (KarthikKumarDev) [inline med-inventory/src/layout/NavBar.jsx:33]: Why do we need this blank line?

File: med-inventory/src/layout/NavBar.jsx [hunk 3]
```diff
@@ -147,6 +183,7 @@ export default function NavBar(props) {
         open={open}
       >
         <div className={classes.toolbar}>
+        <img className={classes.logoImageOpen} src= "/images/siddha.jpg" alt="logo" />
```
Comment (KarthikKumarDev) [inline med-inventory/src/layout/NavBar.jsx]: "logoImageOpen" doesn't give any meaning, please give meaningful names

File: med-inventory/src/layout/NavBar.jsx [hunk 4]
```diff
@@ -82,11 +92,27 @@ const useStyles = makeStyles(theme => ({
...
+  toolbarStart:{
+    display : "flex",
+    alignItems : "start",
+    justifyContent : "flex-start"
+  },
+  toolbarEnd:{
+    display : "flex",
+    alignItems : "end",
+    justifyContent : "flex-end",
+    marginLeft :940
```
Comment (KarthikKumarDev) [inline med-inventory/src/layout/NavBar.jsx]: Margin Left value is hard coded. Does this work well in different screen sizes/resolutions?

**Review by KarthikKumarDev — CHANGES_REQUESTED** [review]

**Comment by KarthikKumarDev** [comment]
image

The white background on blue toolbar doesn't look great. Feel free to change the toolbar color. 
One quick reccomendation is the Icon should be laid on top of the toolbar. Should not have a separate background. For example:
image

**Comment by Kani1996** [comment]
image

**Comment by Kani1996** [comment]
I think the blue tool bar looks different... I have attached a sample screen(Angular)
I will change the logo without white background...

File: med-inventory/src/layout/NavBar.jsx (hunk 4 above)
Comment (Kani1996) [reply med-inventory/src/layout/NavBar.jsx]: I will do it with Flex and update

File: med-inventory/src/layout/NavBar.jsx (hunk 3 above)
Comment (Kani1996) [reply med-inventory/src/layout/NavBar.jsx]: I thought the style represents logo when the drawer is open

File: med-inventory/src/layout/NavBar.jsx (hunk 1 above)
Comment (Kani1996) [reply med-inventory/src/layout/NavBar.jsx:12]: Going forward I will mention all the changes done

File: med-inventory/src/layout/NavBar.jsx (hunk 3 above)
Comment (KarthikKumarDev) [reply med-inventory/src/layout/NavBar.jsx]: Maybe give "open-drawer-logo".Or something that is meaningful

**Comment by karthikkumar1996** [comment]
Also, build is breaking due to this 
`./src/layout/NavBar.jsx
  Line 9:8:  'Typography' is defined but never used  no-unused-vars`
Please take a look at this too

**Comment by KarthikKumarDev** [comment]
- [x] Also, build is breaking due to this
./src/layout/NavBar.jsx Line 9:8: 'Typography' is defined but never used no-unused-vars
Please take a look at this too

**Comment by Kani1996** [comment]
ok I will remove unused imports.

File: med-inventory/src/layout/NavBar.jsx [hunk 5]
```diff
@@ -57,10 +56,7 @@ const useStyles = makeStyles(theme => ({
   menuButton: {
     marginRight: 13
   },
-  logoImage:{
-   height: "64px"
-  },
-  logoImageOpen:{
+  logoDrawerImageOpen:{
```
Comment (KarthikKumarDev) [inline med-inventory/src/layout/NavBar.jsx:59]: Again, here its not image that is open. Use proper names going forward. Closing this for now

**Comment by karthikkumar1996** [comment]
So, how does the UI looks like now? And what does this PR do?

**Comment by KarthikKumarDev** [comment]
https://sidha-med-inventory.firebaseapp.com/

**Pull request merged**